### ISSUE TITLE
Version 0.2.5 broken?

### ISSUE THREAD

**Issue by Nedlinin** [tatsuyafw/gulp-nightwatch#29]
Upgrading to 0.2.5 appears to break my gulp build.  Build fails both on my local dev box and my CI box.  0.2.4 is fine.

This is on NPM 3.3.3 and Node 4.1.1.  

0.2.5 appears not to have an index.js file pulled anymore.

    [10:40:10] Requiring external module babel-core/register

    module.js:338
        throw err;
        ^

    Error: Cannot find module 'gulp-nightwatch'
        at Function.Module._resolveFilename (module.js:336:15)
        at Function.Module._load (module.js:286:25)
        at Module.require (module.js:365:17)
        at require (module.js:384:17)
        at Object.<anonymous> (C:/Users/cheath/Documents/dashboard/gulp/tasks/uitest.js:2:30)
        at Module._compile (module.js:434:26)
        at normalLoader (C:\Users\cheath\Documents\dashboard\node_modules\babel-core\lib\api\register\node.js:199:5)
        at Object.require.extensions.(anonymous function) [as .js] (C:\Users\cheath\Documents\dashboard\node_modules\babel-core\lib\api\register\node.js:216:7)
        at Module.load (module.js:355:32)
        at Function.Module._load (module.js:310:12)

**Comment by davidangel** [comment]
I'm getting the same error on 0.2.5.

**Comment by davidangel** [comment]
Looking at my node_modules directory after installing gulp-nightwatch:
```
    gulp-nightwatch/
        lib
        node_modules/
        .npmignore
        LICENSE
        package.json
        README.md
```
I seem to be missing the index.js and a few other files from what I'm seeing here in the repo...

**Comment by vigneshjeyaraj** [comment]
+1 It is broken for me too.

**Comment by fjl3** [comment]
+1

**Comment by Nedlinin** [comment]
Looks like there is an issue with npm publish which sometimes causes this sort of bug.

https://github.com/npm/npm/issues/5082

Work around for now seems to be simply republishing.

**Comment by tatsuyafw** [comment]
Sorry, everyone...

I've released `v0.2.6`, and confirmed the new version fixed this issue.
Please check it.
